grunt-configs/copy.js

```diff
@@ -20,6 +20,16 @@ module.exports = function (grunt, options) {
                 },
                 {
                     expand: true,
+                    cwd: 'node_modules',
+                    src: [
+                        'react/dist/react-with-addons.js',
+                        'JSON/json2.js'
+                    ],
```

Comment:
would it be possible to locate all the vendor based dependencies from npm? then we could remove `static/src/javascripts/vendor`, and have `formstack`, `prebid` and `stripe` installed from `node_modules`